sys/net/gnrc/pktbuf/include/pktbuf_internal.h

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (C) 2014 Martine Lenders <mlenders@inf.fu-berlin.de>
+ *               2015 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     net_gnrc_pktbuf
+ * @brief       Internal stuff for gnrc_pktbuf
+ *
+ * @{
+ *
+ * @file
+ * @brief   Interface definition for the global network buffer. Network devices
+ *          and layers can allocate space for packets here.
+ *
+ * @author  Martine Lenders <mlenders@inf.fu-berlin.de>
+ * @author  Hauke Petersen <hauke.petersen@fu-berlin.de>
+ */
+#ifndef NET_GNRC_PKTBUF_INTERNAL_H
+#define NET_GNRC_PKTBUF_INTERNAL_H
+
+#include <stdbool.h>
+#include <stdlib.h>
+
+#include "mutex.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Mutex used to provide mutually exclusive access to packet buffer
+ *          data structures
+ *
+ * @warning This mutex will be ***internally*** locked and unlocked and should
+ *          not be touched by external code
+ */
+extern mutex_t gnrc_pktbuf_mutex;
+
+#if IS_USED(MODULE_GNRC_PKTBUF_STATIC) || DOXYGEN
+/**
+ * @brief   The actual static buffer used when module gnrc_pktbuf_static is used
+ *
+ * @warning This is an internal buffer and should not be touched by external code
+ */
+extern uint8_t *gnrc_pktbuf_static_buf;
+#endif
+
+#if IS_USED(MODULE_GNRC_PKTBUF_STATIC) || DOXYGEN
```

Comment:
Can we pull the `IS_USED()` into the function?

Reply:
Done and squashed right in.

Reply:
There are some more functions for which you can do this ;-).